feature/notification/src/main/java/com/iteneum/notification/presentation/NotificationViewModel.kt

```diff
@@ -0,0 +1,71 @@
+package com.iteneum.notification.presentation
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.iteneum.network.DataState
+import com.iteneum.notification.domain.models.NotificationDomain
+import dagger.hilt.android.lifecycle.HiltViewModel
+import kotlinx.coroutines.launch
+import javax.inject.Inject
+
+/**
+ * [NotificationViewModel] is the view model what work with the NotificationListView, here is all the information will be consumed and passed to the view
+ *
+ * @author Usiel Filiberto Garcia Jimenez
+ */
+@HiltViewModel
+class NotificationViewModel @Inject constructor() : ViewModel(){
+
+    var dataToday: MutableList<NotificationDomain> = mutableListOf()
```

Comment:
you need to create a two variables, to manage the data, also you don't follow the Carlos ViewModels rules, please check with Carlos.

Reply:
we can only use 1 list variable and to make a difference between "today" and "week" with a new property in the domain data class